__tests__/app.test.js

```diff
@@ -244,3 +249,31 @@ describe("GET/api/articles/:article_id/comments", () => {
     })
   });
 });
+
+describe('POST/api/articles/:article_id/comments', () => {
+    test('STATUS - 201: should add a new comment to the specified article and return the new comment', () => {
+        return request(app)
+        .post("/api/articles/2/comments")
+        .send({username: "rogersop", body: "Very interesting"})
+        .expect(201)
+        .then((response) =>{
+            expect(response.body.newComment).toMatchObject({
+                comment_id: expect.any(Number),
+                body: "Very interesting",
+                article_id: 2,
+                author: "rogersop",
+                votes: expect.any(Number),
+                created_at: expect.any(String)
+            })
+        })
+    });
+    test('STATUS - 400: Should return an appropriate error when passed an invalid body', () => {
+      return request(app)
+      .post("/api/articles/2/comments")
+      .send({})
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("bad request")
+      })
+    });
+});
```

Comment:
There's quite a few missing tests here. What about posting to article 999, what about posting as a user that is not in the database, what about sending extra keys, what about no comment body.